### PR TITLE
anfog_rt test

### PR DESCRIPTION
**Pull request by gsatimos**
Hi @bpasquer 
I've started putting up separate PRs for each new test. For each one, the things that I generally don't know (or need confirming) are:
- what input file(s) to use to both ADD and UPDATE
- what to delete (if anything)
- whether the tables I've identified are appropriate (and if there's any columns that should be excluded).

Would you mind pushing files any files I need to this branch to `input_files/anfog_rt/`?

### CODE REVIEW

File: pipeline/talend_test/test_configs/anfog_rt.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+talend_log_file: /mnt/ebs/log/pipeline/process/tasks.ANFOG_RT.log
+talend_jobs:
+  - anfog_rt-anfog_rt
+database_schemas:
+  -
+    name: anfog_rt-anfog_rt
+    tables:
+      - name: anfog_rt_trajectory_map
+        exclude_columns: []
+      - name: anfog_rt_trajectory_map
```
Comment (bpasquer) [inline pipeline/talend_test/test_configs/anfog_rt.yaml]: Did you mean `anfog_rt_trajectory_data` here?

**Comment by bpasquer** [comment]
For this harvester, you only need to do an ADD and DELETE.
Yes, I'll push 2 files in `input_files/anfog_rt`

File: pipeline/talend_test/test_configs/anfog_rt.yaml (hunk 1 above)
Comment (gsatimos) [reply pipeline/talend_test/test_configs/anfog_rt.yaml]: I sure did :) sorry, copy/paste error by me.

**Comment by gsatimos** [comment]
thanks @bpasquer 
I've just got an issue trying to trigger this pipeline. Nothing seems to happen when the file arrives in `/mnt/ebs/incoming/ANFOG/processed` and there's nothing in the log output. Is this the correct directory?

**Comment by bpasquer** [comment]
`/mnt/ebs/incoming/ANFOG/processed` is the incoming ANFOG_DM files. 
`/mnt/ebs/incoming/ANFOG/realtime` is the one you want to push anfog rt files to

**Comment by gsatimos** [comment]
Hi @bpasquer 
I'm having a little trouble running this pipeline. The `tasks.ANFOG_RT.log` log produces this:
```
2019-05-09 14:57:20,583 ERROR tasks.ANFOG_RT[1a30f3cd-8b9f-4ad5-86d1-8140480b594e] InvalidInputFileError: Cannot process the uploaded file IMOS_ANFOG_BCEOSTUV_20190425T005939Z_SL210_FV00_timeseries_END-20190430T061605Z.nc.
Traceback (most recent call last):
  File "/mnt/ebs/aodn-pipeline/local/lib/python2.7/site-packages/aodncore/pipeline/handlerbase.py", line 1029, in run
    self.trigger(transition['trigger'])
  File "/mnt/ebs/aodn-pipeline/local/lib/python2.7/site-packages/transitions/core.py", line 33, in get_trigger
    return func(*args, **kwargs)
  File "/mnt/ebs/aodn-pipeline/local/lib/python2.7/site-packages/transitions/core.py", line 283, in trigger
    return self.machine._process(f)
  File "/mnt/ebs/aodn-pipeline/local/lib/python2.7/site-packages/transitions/core.py", line 840, in _process
    return trigger()
  File "/mnt/ebs/aodn-pipeline/local/lib/python2.7/site-packages/transitions/core.py", line 313, in _trigger
    if t.execute(event_data):
  File "/mnt/ebs/aodn-pipeline/local/lib/python2.7/site-packages/transitions/core.py", line 195, in execute
    machine._callback(func, event_data)
  File "/mnt/ebs/aodn-pipeline/local/lib/python2.7/site-packages/transitions/core.py", line 823, in _callback
    func(*event_data.args, **event_data.kwargs)
  File "/mnt/ebs/aodn-pipeline/local/lib/python2.7/site-packages/aodndata/anfog/handlers.py", line 103, in preprocess
    .format(name=input_file_basename))
InvalidInputFileError: Cannot process the uploaded file IMOS_ANFOG_BCEOSTUV_20190425T005939Z_SL210_FV00_timeseries_END-20190430T061605Z.nc.
```
Looks like maybe the filenames are not matching the regex. Any suggestions?

**Comment by bpasquer** [comment]
My mistake @gsatimos, I should have mentioned that anfog_rt data are submitted as a ZIP.

**Comment by gsatimos** [comment]
@jonescc can you review and merge please

**Pull request merged**